SPECS/glib/glib.spec

```diff
@@ -69,18 +71,19 @@ The glib2-doc package includes documentation for the GLib library.
 
 %prep
 %setup -q
-%patch0 -p1
-%patch1 -p1
 
 %build
-./autogen.sh
-%configure --with-pcre=system \
-            --enable-static \
-            --enable-gtk-doc
-make %{?_smp_mflags}
+%meson \
+    -Dgtk_doc=true \
+    --default-library=both
+
+%meson_build
 
 %install
-make DESTDIR=%{buildroot} install
+%meson_install
+
+mkdir -p %{buildroot}%{_libdir}/gio/modules
+touch %{buildroot}%{_libdir}/gio/modules/giomodule.cache
```

Comment:
Why do we create this manually now? What was it used for in the previous versions and is it OK for this file to be empty?

Reply:
In the configuration file of the previous version, there's a section of code that will determine the default path of gio modules directory and that piece of code is gone in the newer version. In the glib spec file of other distributions, it's also manually creating this directory.

In the previous version, we don't even have this cache file. The gio/modules directory is empty.